testing/peervpn/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=peervpn
+pkgver=0.044
+pkgrel=0
+pkgdesc="PeerVPN builds virtual ethernet networks between multiple computers"
+url="https://peervpn.net"
+arch="all"
+license="GPL3"
+depends="openssl zlib"
+makedepends="$depends openssl-dev zlib-dev linux-headers"
```

Comment:
I guess i should aim for the most recent release. Would that be with libressl? Is it possible to create a package that could work with either? This is my first package So i might need some guidance.